.github/workflows/x-build.yaml

```diff
@@ -0,0 +1,66 @@
+# ********************************************************************************
+#  Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  SPDX-License-Identifier: Apache-2.0
+# *******************************************************************************/
+
+# Run all-feature tests on multiple architecture targets
+# Note: tbd whether this adds any actual value...
+
+name: X-platform build
+
+on:
+  workflow_call:
+  workflow_dispatch:
+            
+env:
+  RUST_TOOLCHAIN: ${{ vars.RUST_TOOLCHAIN || 'stable' }}
+  RUSTFLAGS: -Dwarnings
+  CARGO_TERM_COLOR: always  
+
+jobs:
+  build:
+    runs-on: ${{ matrix.runner }}
+    env: 
+      NEXTEST_EXPERIMENTAL_LIBTEST_JSON: 1
+    strategy:
+      matrix:
+        include:
+          - name: linux-amd64
+            runner: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+          - name: linux-arm64
+            runner: ubuntu-latest
+            target: aarch64-unknown-linux-gnu
```

Comment:
:+1: 

Was about to ask for this yesterday on the issue, but you beat me to the punch :slightly_smiling_face: